screens/ReservationUpcomingForm.js

```diff
@@ -280,8 +287,92 @@ class ReservationUpcomingForm extends React.Component {
 
         })
       }).then()
+  }
+
+  handleSearchMember = (phone) => {
```

Comment:
These two functions are duplicated. Is there any way to consolidate them in a single location?